makenrs

```diff
@@ -181,12 +136,17 @@ mkdir -p "${NEKRS_BUILD_DIR}" > /dev/null 2>&1
 rm -rf "${NEKRS_BUILD_DIR}"/CMakeCache.txt "${NEKRS_BUILD_DIR}"/*.cmake > /dev/null 2>&1
 
 cd "${NEKRS_BUILD_DIR}"
-  FC="${NEKRS_FC}" CC="${NEKRS_CC}" CXX="${NEKRS_CXX}" \
-  FFLAGS="${NEKRS_FFLAGS}" CFLAGS="${NEKRS_CFLAGS}" CXXFLAGS="${NEKRS_CXXFLAGS}" \
-  cmake -DCMAKE_BUILD_TYPE="${NEKRS_BUILD_TYPE}" \
-        -DCMAKE_INSTALL_PREFIX=${NEKRS_INSTALL_DIR} \
-        -DNEK5000_PPLIST=${NEKRS_NEK5000_PPLIST} \
-        ${NEKRS_DIR} && make install
+cmake -DCMAKE_BUILD_TYPE="${NEKRS_BUILD_TYPE}" \
+      -DCMAKE_Fortran_COMPILER="${NEKRS_FC}" \
+      -DCMAKE_C_COMPILER="${NEKRS_CC}" \
+      -DCMAKE_CXX_COMPILER="${NEKRS_CXX}" \
+      -DCMAKE_Fortran_FLAGS="${NEKRS_FFLAGS}"  \
+      -DCMAKE_C_FLAGS="${NEKRS_CFLAGS}" \
+      -DCMAKE_CXX_FLAGS="${NEKRS_CXXFLAGS}" \
+      -DCMAKE_INSTALL_PREFIX="${NEKRS_INSTALL_DIR}" \
+      -DNEK5000_PPLIST="${NEKRS_NEK5000_PPLIST}" \
```

Comment:
`${NEKRS_NEK5000_PPLIST}` needed to be in quotes.  When it wasn't in quotes, and the variable had more than one symbol, it wasn't being properly assigned to the cache variable